bin/setenv-docker-customize.sh

```diff
@@ -694,14 +672,6 @@ if [ -f /etc/exo/chat.properties ] && [ "${EXO_CHAT_SERVER_STANDALONE}" = "false
   sed -i 's/^chatPassPhrase=.*$/chatPassPhrase='"$(tr -dc '[:alnum:]' < /dev/urandom  | dd bs=4 count=6 2>/dev/null)"'/' /etc/exo/chat.properties
 fi
 
-
-# -----------------------------------------------------------------------------
-# Configure the eXo profiles for clustering if needed
-# -----------------------------------------------------------------------------
-if [ "${EXO_CLUSTER}" = "true" ]; then
```

Comment:
what would activate the profile cluster in the future?

Reply:
This option is interpreted in product : 
https://github.com/Meeds-io/meeds/blob/d9bcded2b38371342ddba3b634bc777e61065e85/packaging/plf-tomcat-resources/src/main/resources/bin/setenv.sh#L97